_episodes/02-filedir.md

```diff
@@ -11,6 +11,8 @@ objectives:
 - "Translate an absolute path into a relative path and vice versa."
 - "Construct absolute and relative paths that identify specific files and directories."
 - "Demonstrate the use of tab completion, and explain its advantages."
+- "How to navigate between directories."
+- "How to list the contents of a directory."
```

Comment:
In terms of wording, learning objectives should generally be able to complete the sentence "After this episode, you'll be able to ___________".

Reply:
Also, it seems like these are a bit redundant with respect to these existing questions above:

> How can I move around on my computer?
> How can I see what files and directories I have?